pkg/keys/ed25519_test.go

```diff
@@ -18,6 +18,10 @@ type Ed25519Suite struct{}
 
 var _ = Suite(&Ed25519Suite{})
 
+func (Ed25519Suite) TestMarshalEd25519(c *C) {
```

Comment:
Empty test; remove

Reply:
How could I go about testing the `MarshalPublicKey` method. I was considering something like this but I am not sure how I can create a custom verifier.

```go
func (Ed25519Suite) TestMarshalEd25519(c *C) {
	badKeyValue, err := json.Marshal(map[string]string{
		"bad_public": "mal key",
	})
	c.Assert(err, IsNil)

	badKey := &data.PublicKey{
		Type:       data.KeyTypeEd25519,
		Scheme:     data.KeySchemeEd25519,
		Algorithms: data.HashAlgorithms,
		Value:      badKeyValue,
	}

	fmt.Println(badKey)

	verifier := ed25519Verifier{
		PublicKey: data.HexBytes(badKey.Value),
		key:       badKey,
	}
	c.Assert(verifier.MarshalPublicKey(), ErrorMatches, "tuf: error unmarshalling key. *")
}
```

Reply:
The point of these `panic()` calls is that it shouldn't be possible to trigger this case, so it's not going to be straightforward to test :) 

I think the existing test coverage is fine